tests/libvmi/vmi.go

```diff
@@ -36,7 +37,7 @@ type Option func(vmi *kvirtv1.VirtualMachineInstance)
 // New instantiates a new VMI configuration,
 // building its properties based on the specified With* options.
 func New(name string, opts ...Option) *kvirtv1.VirtualMachineInstance {
-	vmi := baseVmi(name)
+	vmi := baseVmi(util.NamespaceTestDefault, name)
```

Comment:
This should not be needed.

Reply:
https://github.com/kubevirt/kubevirt/pull/7551#discussion_r849195120